MissouriFoodBanks/__init__.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import json
+import os
+import sys
+
+import datetime
+from pymongo import MongoClient, TEXT
+import azure.functions as func
+
+from .missouri_food_banks import mfb_scraper
+
+
+def main(mytimer: func.TimerRequest, context: func.Context) -> None:
+    utc_timestamp = datetime.datetime.utcnow().replace(
+        tzinfo=datetime.timezone.utc
+    ).isoformat()
+    conn_string = os.environ['MONGO_DB_CONNECTION_STRING']
+    client = MongoClient(conn_string)['shelter']
+    scraped_update_date = mfb_scraper.scrape_updated_date()
+    stored_update_date = mfb_scraper.retrieve_last_scraped_date(client)
```

Comment:
This code is repeated in here as well as in main method of the scraper. If you observe it closely it's actually same code for every scraper, hence we can move it into it's own method in BaseScraper.py . I have done that already as part of WASHINGTON_DC_Shelters scraper.  Or in ILMS scraper too. Have a look.

Reply:
The message `No new MFB data. Goodbye...` can be generic e.g. `No new data. Goodbye`.

Reply:
so, calling main scraper would be enough right? I can remove lines 19 and 20 bevause it is redundant? Did I understand it right? :)

Reply:
Yes